### ISSUE TITLE
from magpy.gui.metapage import *, ImportError: No module named metapage

### ISSUE THREAD

**Issue by mojca** [geomagpy/magpy#5]
The gui branch doesn't work for me due to a missing file:

~~~
Loading python's SQL support
... success
Traceback (most recent call last):
  File "/opt/local/bin/xmagpy-2.7", line 3, in <module>
    from magpy.gui.magpy_gui import MagPyApp
  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/magpy/gui/magpy_gui.py", line 27, in <module>
    from magpy.gui.metapage import *
ImportError: No module named metapage
~~~

**Comment by leonro** [comment]
Sorry, forget to add those files. They are online now. Still need to run some test and add the multiple stream support. There are still many issues but generally most routines (except MARTAS and MQTT monitoring, as well as all multiple stream things) should work.

Roman

Dr. Roman Leonhardt
Conrad Observatorium
ZAMG - Zentralanstalt für Meteorologie und Geodynamik
1190 Wien, Hohe Warte 38
Tel.: +43 1 36026 2507
Fax: +43 1 36866 21
E-Mail: roman.leonhardt@zamg.ac.at<https://webmail.zamg.ac.at/owa/redir.aspx?SURL=t9AS8jCcPggsx8oPf2D18nQbebRT-hDvZYGVU9uNLKVg80_NrjnSCG0AYQBpAGwAdABvADoAcgBvAG0AYQBuAC4AbABlAG8AbgBoAGEAcgBkAHQAQAB6AGEAbQBnAC4AYQBjAC4AYQB0AA..&URL=mailto%3aroman.leonhardt%40zamg.ac.at>
www.zamg.ac.at<https://webmail.zamg.ac.at/owa/redir.aspx?SURL=DgfuyOdHyWS5QXRRIyzOHmne5_jxCaCmQXd1H4B7AnJg80_NrjnSCGgAdAB0AHAAOgAvAC8AdwB3AHcALgB6AGEAbQBnAC4AYQBjAC4AYQB0AC8A&URL=http%3a%2f%2fwww.zamg.ac.at%2f>


Conrad Observatorium
2763 Muggendorf , Trafelberg 1-2
Tel.: +43 2632 73276 oder 7367221
www.conrad-observatory.at<https://webmail.zamg.ac.at/owa/redir.aspx?SURL=3GEn2GkM89BDePjqOqAYT0d3pi9ZpUlcz8Zli53n9uNg80_NrjnSCGgAdAB0AHAAOgAvAC8AdwB3AHcALgBjAG8AbgByAGEAZAAtAG8AYgBzAGUAcgB2AGEAdABvAHIAeQAuAGEAdAAvAA..&URL=http%3a%2f%2fwww.conrad-observatory.at%2f>
________________________________
Von: Mojca Miklavec [notifications@github.com]
Gesendet: Dienstag, 2. August 2016 15:33
An: geomagpy/magpy
Betreff: [geomagpy/magpy] from magpy.gui.metapage import *, ImportError: No module named metapage (#5)


The gui branch doesn't work for me due to a missing file:
{{{
Loading python's SQL support
... success
Traceback (most recent call last):
File "/opt/local/bin/xmagpy-2.7", line 3, in
from magpy.gui.magpy_gui import MagPyApp
File "/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/magpy/gui/magpy_gui.py", line 27, in
from magpy.gui.metapage import *
ImportError: No module named metapage
}}}

**Comment by leonro** [comment]
Fixed: 1fc61772a92f256e5416486ed8d6a8d78daa5f59
